src/EnergyPlus/DataSizing.hh

```diff
@@ -127,35 +145,28 @@ namespace DataSizing {
     static constexpr std::string_view PeakHrMinFmt("{:02}:{:02}:00");
 
     // Zone Outdoor Air Method
-    constexpr int ZOAM_FlowPerPerson(1); // set the outdoor air flow rate based on number of people in the zone
-    constexpr int ZOAM_FlowPerZone(2);   // sum the outdoor air flow rate per zone based on user input
-    constexpr int ZOAM_FlowPerArea(3);   // sum the outdoor air flow rate based on zone area
-    constexpr int ZOAM_FlowPerACH(4);    // sum the outdoor air flow rate based on number of air changes for the zone
-    constexpr int ZOAM_Sum(5);           // sum the outdoor air flow rate of the people component and the space floor area component
-    constexpr int ZOAM_Max(6);           // use the maximum of the outdoor air flow rate of the people component and the space floor area component
-    constexpr int ZOAM_IAQP(7);          // Use ASHRAE Standard 62.1-2007 IAQP to calculate the zone level outdoor air flow rates
-    constexpr int ZOAM_ProportionalControlSchOcc(8); // Use ASHRAE Standard 62.1-2004 or Trane Engineer's newsletter (volume 34-5)
-    // to calculate the zone level outdoor air flow rates based on scheduled occupancy
     constexpr int ZOAM_ProportionalControlDesOcc(9); // Use ASHRAE Standard 62.1-2004 or Trane Engineer's newsletter (volume 34-5)
     // to calculate the zone level outdoor air flow rates based on design occupancy
 
+    enum class SOAM
```

Comment:
I would rather call this `SystemOAMethod`. I would not have guessed what `SOAM` is.